lib/internal/Magento/Framework/Serialize/Serializer/Json.php

```diff
@@ -34,7 +34,13 @@ public function serialize($data)
      */
     public function unserialize($string)
     {
+        if ($string === null) {
+            throw new \InvalidArgumentException(
+                sprintf('Invalid parameter: string must be a string type, null given.')
```

Comment:
`Unable to unserialize value. Error: Parameter must be a string type, null given.` Could be a better option for message to keep aligned with existing pattern

Reply:
@xmav, I changed message by your recommendation

Thank you.